salt/modules/dockermod.py

```diff
@@ -1442,6 +1442,92 @@ def login(*registries):
     return ret
 
 
+def logout(*registries):
+    '''
```

Comment:
JFYI: https://github.com/saltstack/salt/commit/0b2a5613b345f17339cb90e60b407199b3d26980

Reply:
Thanks for the reminder. I'll run `black` over the change to make possible future backports easier.